src/models/bookDao.js

```diff
@@ -0,0 +1,46 @@
+const appDataSource  = require("./dataSource");
+
+const getAllBooks = async () => {
+  const resultAll = await appDataSource.query(
+    `
+		SELECT 
+      id,
+      sub_category_id,
+      title,
+      image_url,
+      published_date,
+      offline_price,
+      online_price,
+      author,
+      publisher,
+      description,
+      publisher_review,
+      list,
+      created_at
+    FROM books
+		`
+  );
```

Comment:
- 우선 들여쓰기 수정해주세요!
- 컬럼명을 그대로 가져오게되면 snake_case로 필드를 반환합니다. 프론트엔드 분들과 소통하여 이 부분 통일하면 좋을 것 같습니다.

```suggestion
  const resultAll = await appDataSource.query(`
    SELECT 
      id,
      sub_category_id, // as subCategoryId 혹은 snake_case 유지  
      title,
      image_url,
      published_date,
      offline_price,
      online_price,
      author,
      publisher,
      description,
      publisher_review,
      list,
      created_at
    FROM books
    `);
```